### ISSUE TITLE
MySQl

### ISSUE THREAD

**Issue by CamTosh** [Cclleemm/FriendlyTorrent#13]
Please use PDO...

Mysql_connect is deprecated

**Comment by Cclleemm** [comment]
Yes, sorry, it's an old code... If someone can adapt it, I could accept the pull request.
I'm so lazy to implement PDO...
